pkg/daemon/daemon.go

```diff
@@ -461,6 +464,17 @@ func (dn *Daemon) applyNodePtpProfile(runID int, nodeProfile *ptpv1.PtpProfile)
 			configFile = fmt.Sprintf("ts2phc.%d.config", runID)
 			configPath = fmt.Sprintf("/var/run/%s", configFile)
 			messageTag = fmt.Sprintf("[ts2phc.%d.config:{level}]", runID)
+		case syncEProcessName:
+			configOpts = nodeProfile.Synce4lOpts
+			if configOpts == nil {
```

Comment:
This means that if a ptpconfig doesn't enable synce manually, it will still run synce.  This is the case with ptp4l, in order to be backwards compatible with other ptpconfigs, but for synce, if we don't specify Synce4lOpts(and so it's nil), then we should just not run synce.

Reply:
Right , I will fix this. Thanks